### ISSUE TITLE
NTR: has allelopath

### ISSUE THREAD

**Issue by mdrishti** [oborel/obo-relations#809]
The following term request is for the inverse interaction of 'allelopath of' http://purl.obolibrary.org/obo/RO_0002555. Linked to https://github.com/globalbioticinteractions/globalbioticinteractions/issues/993#issuecomment-2291292052

## Preferred term label

has allelopath

## Synonyms

none

## Textual definition

A relationship between organisms where one organism is influenced by the biochemical processes of another.

Additional context: 
Allelopathy is a process whereby biochemicals produced by one organism positively or negatively influences growth, survival or reproduction of other organisms. See also https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10172429/

## Suggested parent term

(http://purl.obolibrary.org/obo/RO_0002321) -> (http://purl.obolibrary.org/obo/RO_0002437) -> (http://purl.obolibrary.org/obo/RO_0002574)

## Attribution

Disha Tandon (https://orcid.org/0009-0005-5515-1230)

**Comment by nataled** [comment]
Not an expert here, but just wondering if the second paragraph under Textual Definition is intended as part of the definition, or is just there to explain allelopathy to us non-experts? My understanding of how definitions are supposed to work is that such explanatory sentences are better as 'comment' (but I do very often see the two mixed).

**Comment by mdrishti** [comment]
@nataled , yes in this case, the 2nd paragraph the textual definition provides additional context behind allelopathy. I plan to put this definition under 'see also' or 'comment, while describing this term.

I added it because allelopathy IMO is a broad area and in fact, in some fields like plant biology it has been thought to be a negative influence of one organism's biochemicals on another, which is not exactly right. Hence, the definition.

**Comment by nataled** [comment]
Thanks for the clarification about the clarification ;)

I'm a bit confused by your followup statement regarding how the term has been (mis?)used in plant biology. I see a few different ways to interpret that statement:

1) Plant biologists use 'allelopathy' to refer specifically to negative influences, but it should actually be positive influences.
2) Plant biologists use 'allelopathy' to refer specifically to negative influences, but it should actually be positive OR negative influences.

I doubt anything that has exclusively a positive influence would be labeled with -pathy, so I'm guessing the second interpretation is correct? If so, I'd recommend that explanatory sentence be revised to say that explicitly:

"Allelopathy is a process whereby biochemicals produced by one organism positively or negatively influences growth, survival or reproduction of other organisms."

Obviously, if I'm incorrect, ignore my suggestion! But I think at least the word 'grow' should be 'growth'?

**Comment by nataled** [comment]
I reviewed the changes made and see that they address my concerns.

**Comment by mdrishti** [comment]
@bpeters42, I will create a pull request soon. Just short of time these days.

**Comment by cmungall** [comment]
Thanks everyone!

Just a general comment to help make us more responsive to community members like @mdrishti , that we in RO shouldn't require requesters to supply definitions for inverse terms, these are all generated.

(If someone could add documentation for our process that would be great! See the sparql in #51.)

Of course, we welcome any suggestions to modifications from @mdrishti  to the definition for the 'canonical' direction!

It looks like the definition here for the canonical direction accidentally got copied to the seeAlso field rather than the definition field, sigh.. http://purl.obolibrary.org/obo/RO_0002555

**Comment by mdrishti** [comment]
@bpeters42, I am confused about how to assign the RO id to this term. When I first made changes to this file ro-edit.owl file (see here: mdrishti/obo-relations@35387984f7b551c46867e654c1415c1c42a28a83 , the id got assigned as RO_0000001, which I assume is not the right way to go. I am new to ontologies and vocabularies, so I will appreciate any help in this regard.

**Comment by ddooley** [comment]
Nov 4, 2024 call: @mdrishti people can be assigned ranges of ID, and this is held in https://github.com/oborel/obo-relations/blob/master/src/ontology/ro-idranges.owl file (or perhaps for one-offs, one can get an ID from another editor with an assigned range).  This pull request already has a good ID assigned for it.  But we can now simplify the definition to be in the pattern that RO has for inverse relations: "inverse of allelopath of".  So any improvement of definition or comments should be placed on the main "alleopath of" relation.  Can you modify the pull request accordingly?

**Comment by mdrishti** [comment]
@ddooley, yes, eventually I figured the concept of ID ranges. So, I added mine with the previous pull request 

I made the changes  suggested by you to the pull request now. Let me know if this looks good. Thanks.
